lib/iex/cloud/connection.rb

```diff
@@ -12,10 +12,10 @@ def connection
 
         options[:headers] = {}
         options[:headers]['Accept'] = 'application/json; charset=utf-8'
+        options[:headers]['Content-Type'] = 'application/json; charset=utf-8'
```

Comment:
Btw, what breaks without this?

Reply:
POST requests don't work without it. 

If remove `to_json` request body serializes as  `isin=US0378331005&token=test-iex-api-secret-token` and server cannot read the token. 

If remove `options[:headers]['Content-Type'] = 'application/json; charset=utf-8'` then server doens't process body as json and cannot read the token too. 

If remove both `invalid (non-String) body of class Hash` raises.